chapters/annotations.tex

```diff
@@ -426,6 +446,8 @@ \section{Symbolic Processing}\label{annotations-for-symbolic-processing}\label{s
 For a parameter, \lstinline!Evaluate = false! ensures that the parameter is a non-evaluable parameter according to \cref{component-variability} (meaning it is not allowed to be used where an evaluable expression (\cref{evaluable-expressions}) is expected).
 For both parameters and constants -- even when the value can be determined during translation -- the model developer further proposes to not utilize the value for symbolic processing.
 
+If the annotation is missing for a parameter or constant the evaluation of the component is tool-dependent.
+
```

Comment:
Isn't this implicit in the general rules for component-annotations defined as effects for specific values?

Reply:
I wouldn't say that it is clear that it is implicitly tool-dependent.